crypto/crypto_aes.c

```diff
@@ -57,7 +59,7 @@ useaesni(void)
 	static int aesnigood = -1;
 	uint8_t key[32];
 	uint8_t ptext[16];
-	size_t i;
+	uint8_t i;
```

Comment:
My mind boggles that that this is considered "saner", but yes, `& 0xff` works, and I'll include that when I update this PR.